src/chainstate/burn/db/processing.rs

```diff
@@ -152,13 +152,13 @@ impl<'a> SortitionHandleTx<'a> {
         let mut next_pox = parent_pox;
         if let Some(ref next_pox_info) = next_pox_info {
             if next_pox_info.is_reward_info_known() {
-                info!(
+                debug!(
```

Comment:
These are nice as `info` logs, right? Reward cycle starts are fairly rare.

Reply:
Yeah we definitely have logs for new reward cycles `sortdb.rs:2857`, I believe.

Reply:
Hmm -- I think at the very least though, we should have an `info` log for when we start a reward cycle _without_ the anchor block.

Reply:
Sounds good will add

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2234/commits/445e25c5863951ce0606d59b4cf66b08529bf801